lnst/Recipes/ENRT/TrafficControlRecipe.py

```diff
@@ -18,17 +18,16 @@ class TcRecipeConfiguration(RecipeConf):
     pass
 
 
-class TrafficControlRecipe(PerfRecipe):
+class TrafficControlRecipe(TrafficControlReq, PerfRecipe):
     """
     Recipe to evaluate the performance of `tc filter` rule installs
 
     Primarily targeted towards testing mlx cards which support hardware and software steering
     """
 
     driver = StrParam(default="mlx5_core")
-
-    host1 = HostReq()
-    host1.eth0 = DeviceReq(label="net1", driver=RecipeParam("driver"))
+    nic_speed = StrParam()
+    nic_model = StrParam()
```

Comment:
this should now be removed since we have the base common class for these parameters

Reply:
Good catch, fixed.